test.py

```diff
@@ -0,0 +1,21 @@
+import unittest
+
+from bouncer import Bouncer, SecretManagerException
+
+
+class TestBouncer(unittest.TestCase):
+
+    def setUp(self):
+        pass
+
+    def test_valid_secret_name(self):
+        bouncer = Bouncer('commons/dev/whitelist')
+        self.assertTrue(bouncer.is_authorized('donotdelete@testemail.com'))
+
+    def test_invalid_secret_name(self):
+        bouncer = Bouncer('jeeze/oh/pete/i/hope/no/one/actually/would/use/this/as/a/secret/name')
+        self.assertRaises(SecretManagerException, bouncer.is_authorized, 'donotdelete@testemail.com', )
+
+    def test_unauthorized_email(self):
+        bouncer = Bouncer('commons/dev/whitelist')
+        self.assertFalse(bouncer.is_authorized('this.email.address.will.not.be.in.whitelist@foo.bar'))
```

Comment:
It would also be good to add a test for the case where the secret name exists in secret manager, but the "email" key does not. Mainly I want to check that if the user setup the secret incorrectly in secret manager, without the "email" key, the error message is sufficiently clear for the user to identify what the problem is and fix it.

Reply:
good call.